packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
+    border: 1px solid #e0e8ed;
+    color: #012159;
+    label {
+      span {
+        display: block;
+        padding: 10px;
+        margin-bottom: 1px;
+        &:hover {
+          background: #e9f8ff;
+        }
+      }
+      input {
+        display: none;
+        &:checked + span {
+          background: #e9f8ff;
+        }
+      }
+    }
+  }
+  .transfer-buttons {
+    width: 10%;
+    float: left;
+    margin-top: 90px;
+
+    .left, .right {
+      display: block;
+      text-align: center;
+      border: 1px solid #e0e8ed;
+      background-color: rgba(#324E68, .8);
+      color: #fff;
+      padding: 10px;
+      margin: 10px;
+      &:hover {
+        cursor: pointer;
+        background-color: #324E68;
+      }
+    }
+  }
+  ul {
+    margin: 0;
+    list-style: none;
+    padding-left: 0;
+    height: 250px;
+    overflow-y: auto;
+  }
+  h1 {
```

Comment:
Isnt this in the lib already? Also H1 is for a page, not a component, and there can be only 1.

Reply:
Have we documented that decision somewhere? I don't recall it being made.

Reply:
It's normal (not gonna say best practice) to have a single H1 in a page to help screen reader navigate it and its role is to state the pages purpose. Also why would you put a H1 in a component?

Reply:
Because html elements such as `article` and `section` (which I realise now, these are not - my bad, thought I'd made it a `section`) reset the document flow. I've always personally been of the opinion that every time you use one of these elements, you should start again with a `h1`, especially in components as you have no idea where in the document flow the component is going to be used and using a `h3` may not make sense. 

Again though, I'm not in charge so I'm happy to go with the flow here. I'd be interested in know what @hxltrhuxze and @xdobtqletrmn 's opinion on this is though.